app/test_split_integer.py

```diff
@@ -2,20 +2,23 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    result = split_integer(8, 3)
+    assert sum(result) == 8
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    result = split_integer(6, 2)
+    assert result == [3, 3]
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    result = split_integer(8, 1)
+    assert result == [8]
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal() -> None:
-    pass
+    assert sorted(split_integer(17, 4)) == [4, 4, 4, 5]
 
 
 def test_should_add_zeros_when_value_is_less_than_number_of_parts() -> None:
-    pass
+    assert sum(split_integer(32, 6)) == 32
```

Comment:
The test `test_should_add_zeros_when_value_is_less_than_number_of_parts` only checks if the sum of the parts equals the original value. It should also verify that the returned array is sorted and that the difference between the max and min numbers is <= 1, as specified in the task requirements.